global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.0",
+    "version": "8.0.100",
```

Comment:
⚠️ We recommend disabling this Renovate update. Example: https://github.com/bitwarden/sdk/pull/294 -- you want to keep this at `8.0.100`.